generators/server/templates/src/main/java/package/gateway/responserewriting/SwaggerBasePathRewritingFilter.java.ejs

```diff
@@ -95,7 +103,12 @@ public class SwaggerBasePathRewritingFilter extends SendResponseFilter {
                 LinkedHashMap<String, Object> map = this.mapper.readValue(response, LinkedHashMap.class);
 
                 String basePath = requestUri.replace("/v3/api-docs", "");
-                map.put("basePath", basePath);
+                String serversUrl = (String) ((List<Map<String, String>>) map.get(SERVERS)).get(0).get(URL);
```

Comment:
add the new tests, and then change the logic here

Reply:
ok